java/AxivityReader.java

```diff
@@ -348,5 +360,25 @@ private static LocalDateTime cwaHeaderLoggingStartTime(ByteBuffer buf) {
     }
 
 
+    private static void setTimeZone(String timeZone) {
+        zoneId = ZoneId.of(timeZone);
+        rules = zoneId.getRules();
+    }
+
+
+    private static void setSessionStart(LocalDateTime ldt) {
+        sessionStart = ldt.atZone(zoneId);
+        sessionStartDST = rules.isDaylightSavings(sessionStart.toInstant());
+    }
```

Comment:
These are called when we start reading a CWA file. They set the timezone and register the DST status of the session start time.